source/vault/ec2_test.go

```diff
@@ -8,16 +8,16 @@ import (
 
 func TestGetNonce(t *testing.T) {
 	filename := "rumpelstilzchen"
-	contents, err := getNonce(filename)
+	_, err := getNonce(filename)
```

Comment:
All of this can probably disappear, as per my previous comment.

Reply:
This followed getNonce swiftly to /dev/null.